qutebrowser/browser/hints.py

```diff
@@ -748,6 +760,13 @@ def start(self, rapid=False, group=webelem.Group.all, target=Target.normal,
         self._context.tab.find_all_elements(selector, self._start_cb,
                                             only_visible=True)
 
+    def current_mode(self):
+        """ Returns the currently active hinting mode (or None otherwise). """
+        if not self._context or not self._context.hint_mode:
+            return None
+
+        return self._context.hint_mode
```

Comment:
I think you could simplify this a bit, and make the check for `None` more explicit:

```python
if self._context is None:
    return None
return self._context.hint_mode
```

On the other hand, will this ever be called when `self._context` is `None`? It only gets called by `HintKeyParser` (i.e., in hint mode), so `self._context` should always be set, no?